batch/sql/estimated-current.sql

```diff
@@ -351,13 +394,16 @@ CREATE TABLE IF NOT EXISTS `regions` (
 
 CREATE TABLE IF NOT EXISTS `batch_attributes` (
```

Comment:
OK, again, confirm for me:

> This is now a poorly named table: it's really `job_group_attributes`. A subsequent PR will rename this table.

And also confirm for me:

> The attributes on the root group id are displayed in the UI as the batch's attributes.

Reply:
Agreed the table name sucks (all of the new tables that have been updated have bad names). Figuring out how to do the rename online will require trying to write to both the old and new table name. I don't think we can just wait for the container to redeploy and I don't want to copy the tables completely. Writing to both the old and new doesn't seem so bad if we build some infrastructure for this. I just haven't thought about it deeply yet.

Nothing has changed about the UI. Since the job group id is always "0" and is not a primary key (yet), it's just as if there's a dummy extra column right now. Shouldn't change any existing queries.

Reply:
Just take batch offline long enough to do the renames. Renames should be near instantaneous. We have no SLAs and even if we did we wouldn't have 100% uptime! 😄

Reply:
Is it okay if we do the renames in a separate PR? I don't want to do too much in one set of changes and there has to be a separate follow up PR to change the foreign key constraints and primary keys as well. We'll also want to rename the indices on those tables.

Reply:
See #13810 for table renames